lib/modulesync/git.rb

```diff
@@ -46,12 +46,13 @@ def self.switch_branch(repo, branch)
     end
 
     def self.pull(git_base, name, branch, project_root, opts)
+      puts "Syncing #{name}"
       Dir.mkdir(project_root) unless Dir.exist?(project_root)
```

Comment:
This can be dropped if you're going to pass in name as `dir/subdir` because you're going to rely on git to create a directory anyway.